tsconfig-common.json

```diff
@@ -2,6 +2,10 @@
 /* Visit https://aka.ms/tsconfig to read more about this file */
 {
     "compilerOptions": {
+        "typeRoots": [
+            "./node_modules/@types",
+            "./types"
+        ],
```

Comment:
**note:** Copied from webapp. Not sure it's necessary, but hasn't caused any harm.